docs/token-burning.md

```diff
@@ -0,0 +1,19 @@
+# Burning Tokens
+
+Burning tokens on Ergo is done similarly to [minting](./transaction-building.md#step-4-2-mint-a-token). As to mint, you need to add an inexistent token to outputs; to burn, you must add tokens as inputs and omit the number of tokens you want to burn from outputs.
+
+Having such an implicit burning mechanism makes it easy to accidentally burn tokens. So that Fleet only allows burning through the `TransactionBuilder.BurnTokens()` method or by explicitly allowing it through `TransactionBuilder.configure(x => x.allowTokenBurning(true))` if a manual approach is needed.
```

Comment:
```suggestion
To protect against accidently burning tokens, Fleet checks to make sure inputs and outputs are equal. To burn tokens, the `TransactionBuilder.BurnTokens()` method is used to declare the tokenId and amount to burn. 
```

Reply:
Thought it was cleaner to include the code for `TransactionBuilder.BurnTokens()` method right after mentioning it, then that you change the initial configuration to remove this protection.